mlir/tools/mlir-tblgen/OpPythonBindingGen.cpp

```diff
@@ -260,11 +261,16 @@ constexpr const char *attributeDeleterTemplate = R"Py(
     del self.operation.attributes["{1}"]
 )Py";
 
-constexpr const char *regionAccessorTemplate = R"PY(
+constexpr const char *regionAccessorTemplate = R"Py(
   @builtins.property
   def {0}(self):
     return self.regions[{1}]
-)PY";
+)Py";
+
+constexpr const char *valueBuilderTemplate = R"Py(
+def {0}({2}) -> {4}:
```

Comment:
As mentioned above I think version still in use is 3.7, if you could check with 3.7 that would be great (I had to drop some type annotations last time).

Reply:
Discussed offline (type hints evidently supported since various `_ext.py`s employ them)